lib/malwareScanner.js

```diff
@@ -17,16 +17,19 @@ async function scanRequest(Attachments, key) {
 
   let currEntity = draftEntity == undefined ? activeEntity : draftEntity
 
-  await updateStatus(AttachmentsSrv, key, "Scanning", currEntity, draftEntity, activeEntity)
-
   if (!scanEnabled) {
-    setTimeout(async () => {
-      DEBUG?.('Malware scanning is disabled. Setting scan status to Clean.')
-      await updateStatus(AttachmentsSrv, key, "Clean", currEntity, draftEntity, activeEntity)
-    }, 5000)
-
-    return
+    if (cds.env.profiles.length === 1 && cds.env.profiles[0] === "development") {
```

Comment:
Can we have more than one profile at one time?

Reply:
In hybrid, `cds.env.profiles` has [hybrid, development]